### PR TITLE
TINY-8594: Backport - Fixed inline toolbars flickering when switching between editors

### PR DESCRIPTION
**Pull request by tiny-ben-tran**
Related Ticket: TINY-8594
Description of Changes:
* Reverted changes that were made in #6916
* Bumped package.json version to 5.10.4

The issue occurring in a single frame makes it difficult to write a reliable test

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] ~Branch prefixed with `feature/` for new features (if applicable)~

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

**Review by lnewson — CHANGES_REQUESTED** [review]
As per the other PR, this needs tests (it really shouldn't have been logged yet given that had been flagged already but it's not going to hurt anything).

File: modules/tinymce/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Inline toolbars flicker when switching between editors #TINY-8594
```
Comment (lnewson) [inline modules/tinymce/CHANGELOG.md:10]: You also need a changelog entry for TINY-8503 if this is meant to fix that as well.

Comment (lnewson) [reply modules/tinymce/CHANGELOG.md]: @tiny-ben-tran the wrong thing was done here... TINY-8594 and TINY-8504 might be caused by the same underlying issue, but they are still different issues and need different changelog entries. TINY-8504 for example was about multiple toolbars rendering when using `fixed_toolbar_container` in inline mode.

Comment (tiny-ben-tran) [reply modules/tinymce/CHANGELOG.md]: You mean TINY-8503 ticket? 
I think so too. I will ask Andy to verify `also fixed TINY-8503`.

Comment (lnewson) [reply modules/tinymce/CHANGELOG.md]: Yes sorry, hit the wrong key.

File: modules/tinymce/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Inline toolbars flicker when switching between editors #TINY-8594 #TINY-8503
```
Comment (lnewson) [inline modules/tinymce/CHANGELOG.md:10]: This still needs fixing per https://github.com/tinymce/tinymce/pull/7748#discussion_r850085979

Comment (tiny-ben-tran) [reply modules/tinymce/CHANGELOG.md]: I added a separated changedlog entry for TINY-8503

**Review by lnewson — APPROVED** [review]

**Review by ltrouton — APPROVED** [review]

**Pull request merged**